Examples/Io/NuclearInteractions/include/ActsExamples/Io/NuclearInteractions/detail/NuclearInteractionParametrisation.hpp

```diff
@@ -0,0 +1,191 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020-2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "ActsExamples/EventData/ExtractedSimulationProcess.hpp"
+#include "ActsExamples/EventData/SimParticle.hpp"
+
+#include <unordered_map>
```

Comment:
Is `unordered_map` actually used in this header? I think you are missing includes for `tuple` and `utility` (for `std::pair`).

Reply:
Deprecated include - fixed

Reply:
But it seems that it needs to included in the `.cpp` file.